docs/references/backend/sessions/authenticate-request.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: authneticateRequest() | User Object
+description: Networkless method to verify a token passed from the frontend.
+---
+
+# `authenicateRequest()`
+
+Method to authenicate a token passed from the frontend. Networkless if the `secretKey` or `jwtKey` are provided, otherwise performs network call to retreieve the JWKS from Clerk's Backend API.
```

Comment:
```suggestion
Authenticates a token passed from the frontend. Networkless if the `secretKey` or `jwtKey` are provided. Otherwise, performs a network call to retrieve the JWKS from Clerk's Backend API.
```